test/framework/easyconfig.py

```diff
@@ -4800,60 +4800,62 @@ def test_det_copy_ec_specs(self):
             return
 
         # use fixed PR (speeds up the test due to caching in fetch_files_from_pr;
-        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/8007
-        from_pr = 8007
-        arrow_ec_fn = 'Arrow-0.7.1-intel-2017b-Python-3.6.3.eb'
-        bat_ec_fn = 'bat-0.3.3-intel-2017b-Python-3.6.3.eb'
-        bat_patch_fn = 'bat-0.3.3-fix-pyspark.patch'
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/22365
+        from_pr = 22365
```

Comment:
@branfosj This is not the best pick, since the contents of this specific PR will change once https://github.com/easybuilders/easybuild-easyconfigs/pull/22345 is merged...

Can we find another one?

Reply:
I cannot find a 5.0.x easyconfig PR with a patch.